src/main/java/ui/page/ProductPage.java

```diff
@@ -0,0 +1,26 @@
+package ui.page;
+
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class ProductPage {
+    private SelenideElement addToCartButton = $(By.xpath("//p[@id='add_to_cart']"));
+    private SelenideElement closePopupIcon = $(By.xpath("//span[@class='cross']"));
+    private SelenideElement cartLable = $(By.cssSelector(".shopping_cart > a"));
+
```

Comment:
Need to rewrite without "By.". "By." it's usage of selenium, not selenide.
Example:
$(".shopping_cart > a")